draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1021,7 +1024,12 @@ timestamp of the previous update.
 
 `tree_head` contains the latest tree head information (see {{tree_head}}).
 
-`signature` is a signature over the encoded `tree_head` field.
+`algorithm` indicates the signature scheme used to compute the signature.  The
+SignatureScheme enumeration is defined in [I-D.ietf-tls-tls13].
+
+`signature` is the value of a signature over the encoded `tree_head` field,
```

Comment:
I suggest: s/is the value of a signature/is computed/

I don't think it's necessary to say "encoded".